OrbitCaptureClient/CaptureEventProcessor.cpp

```diff
@@ -64,6 +64,9 @@ void CaptureEventProcessor::ProcessEvent(const CaptureEvent& event) {
     case CaptureEvent::kTracepointEvent:
       ProcessTracepointEvent(event.tracepoint_event());
       break;
+    case CaptureEvent::kModulesUpdateEvent:
+      // TODO (http://b/168797897): Process module update events
```

Comment:
Should we maybe log something in this case?

Reply:
I am addressing it in the next change. But still - will do.

Reply:
Have you considered adding a test for that already? :)